unpackaged/config/release_gating/customMetadata/Product_Registry.EDADemoReleaseGate.md-meta.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please rename this file to `___NAMESPACE___Product_Registry.EDADemoReleaseGate.md-meta.xml`

This will alow us to deploy it in a managed context.

Reply:
Done, renamed with the namespace prefix